### PR TITLE
Mobile Release v1.91.0

### PR DESCRIPTION
**Pull request by SiobhyB**
## Description
Release 1.91.0 of the react-native-editor and Gutenberg-Mobile.

For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5564

### CODE REVIEW

File: packages/react-native-editor/CHANGELOG.md
Comment (fluiddot) [inline packages/react-native-editor/CHANGELOG.md:1]: I checked the PRs with the label `Mobile App - i.e. Android or iOS` that got merged after the release `1.90.0` ([reference](https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+is%3Aclosed+label%3A%22Mobile+App+-+i.e.+Android+or+iOS%22)), and noticed that some of them are not included in the changelog:

Screenshot 2023-03-16 at 15 13 44

I'm wondering if we could add them, @SiobhyB WDYT?

Comment (fluiddot) [reply packages/react-native-editor/CHANGELOG.md:1]: I'd like to note that https://github.com/WordPress/gutenberg/pull/48776 was superseded by https://github.com/WordPress/gutenberg/pull/48802, so we could add a single entry for both.

**Review by fluiddot — APPROVED** [review]
LGTM 🎊 !

I just added a minor comment about potential new entries in the changelog but shouldn't block the PR.

File: packages/react-native-editor/CHANGELOG.md
Comment (SiobhyB) [reply packages/react-native-editor/CHANGELOG.md:1]: @fluiddot, I'd opted not to include them as they're not user-facing, but could certainly add them if you feel they should be included. :)

Comment (fluiddot) [reply packages/react-native-editor/CHANGELOG.md:1]: Yep, good point. Both PRs although they introduce changes in the user interface shouldn't be noticeable to the user so let's not include them 👍. Hence, the changelog is ok as is, thanks!

**Pull request merged**